micro-rdk/src/esp32/entry.rs

```diff
@@ -205,7 +206,11 @@ pub async fn serve_web_inner(
         .with_webrtc(webrtc)
         .with_periodic_app_client_task(Box::new(RestartMonitor::new(|| unsafe {
             crate::esp32::esp_idf_svc::sys::esp_restart()
-        })));
+        })))
+        .with_periodic_app_client_task(Box::new(ConfigMonitor::new(||unsafe {
+            crate::esp32::esp_idf_svc::sys::esp_restart()
+        },  *(cfg_response.clone()))))
+        ;
```

Comment:
Formatting looks a little off, could you please run `cargo fmt`? I also recommend configuring your editor to do it for you automatically.